Core/StarlightItem.Events.cs

```diff
@@ -65,14 +65,48 @@ public override bool CanUseItem(Item Item, Player Player)
             return true;
         }
 
-		public override void Unload()
+        public delegate bool CanEquipAccessoryDelegate(Item item, Player player, int slot, bool modded);
```

Comment:
Does ModItem not already have hooks for this? This seems redundant

Reply:
I'll let these be since there may be additional use here if we need to design something such as armor which prevents the use of an accessory